src/app/lib/config/services/gahuza.js

```diff
@@ -142,31 +143,36 @@ const service = {
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'Read about our approach to external linking.',
+        text: "Soma ibijanye n'aho duhagaze ku mihora ijana ahandi",
       },
       links: [
         {
-          href: 'https://www.bbc.com/news/help-41670342',
-          text: 'Why you can trust the BBC',
+          href: 'https://www.bbc.com/gahuza/institutional-49283343',
+          text: 'Igituma ushobora kwizera BBC News',
+        },
+        {
+          href: 'https://www.bbc.com/gahuza/institutional-35754053',
+          text: 'Ingingo zo gukoresha urubuga',
         },
         {
-          href: 'https://www.bbc.com/terms',
-          text: 'Terms of Use',
+          href: 'https://www.bbc.com/gahuza/institutional-35754055',
+          text: 'Ibijanye na BBC',
         },
         {
-          href: 'https://www.bbc.com/privacy/',
-          text: 'Privacy Policy',
+          href: 'https://www.bbc.com/gahuza/institutional-35754059',
+          text: 'Ibigenga ubuzima bwite',
```

Comment:
```suggestion
          text: 'Ibigenga n'ubuzima bwite',
```
As per slack conversation, we should match what's on live (and in the footer tab of the translations) rather than the main tab.